controller/rest/crdvalidatewebhook.go

```diff
@@ -200,6 +202,7 @@ func (q *tCrdRequestsMgr) writeCrOpEvent(kind, recordName, uid string, ev share.
 // Third it will call process. if failed a crd delete will issued to remove from k8s
 func (q *tCrdRequestsMgr) crdQueueProc() {
 	var recordList map[string]*share.CLUSCrdSecurityRule
+	//nolint:gosimple
```

Comment:
added.

Reply:
Thanks it's very helpful.  I think the problem linter complained about is that this for loop never ends?  Normally there should be another channel to terminate this loop.  For example,

```
    for {
        select {
        case t := <- timer.C:
            // Do something
        case <-done:
            break loop
        }
    }
```

It probably also makes sense to add a `shutdown()`, so it can be called when controller shuts down. 

```
type tCrdRequestsMgr struct {
        ...
        done           chan int
}

func (q *tCrdRequestsMgr) showdown() {
    close(q.done)
}
```